pages/participantinfo.tsx

```diff
@@ -0,0 +1,476 @@
+                        >
+                            Skip for Now
+                        </Button>
+                    </HStack>
+                </Box>
+            );
+        }
+        return (
+            <FormButton onClick={() => setPageNum((prevPage) => prevPage + 1)}>
+                Next
+            </FormButton>
+        );
+    };
+
+    const medicationDetails =
+        isOnMedication === RADIO_YES ? (
+            <Box>
+                <FormControl id="details">
+                    <FormLabel>
+                        Please provide any details if necessary{" "}
+                    </FormLabel>
+                    <Input placeholder="Details" />
+                </FormControl>
+            </Box>
+        ) : null;
+
+    const allergyDetails =
+        hasAllergies === RADIO_YES ? (
+            <Box>
+                <FormControl id="details">
+                    <FormLabel>
+                        Please provide any details if necessary{" "}
+                    </FormLabel>
+                    <Input placeholder="Details" />
+                </FormControl>
+            </Box>
+        ) : null;
+
+    const formPageHeaders = [
+        "Participant Information ",
+        "Participant Information",
+        "Parent Guardian Information",
+        "Participant Emergency Form",
+        "Participant Health Form",
+        "Confirm Participants",
+        "Proof of Income",
+        "How did you hear about us?",
+    ];
+
+    const formPages = [
+        <Stack spacing={8}>
+            <Box maxW="55rem">
+                <Text noOfLines={2} fontSize="16px" fontWeight="200">
+                    Please provide information on the participant that is being
+                    registered in the program. An opportunity to add information
+                    of additional participants you would like to register will
+                    be provided afterwards.
+                </Text>
+            </Box>
+            <FormLabel>
+                {" "}
+                Emergency Contact Name
+                <HStack spacing="24px">
+                    <FormControl id="first-name">
+                        <Input placeholder="First name" />
+                    </FormControl>
+                    <FormControl id="last-name">
+                        <Input placeholder="Last name" />
+                    </FormControl>
+                </HStack>
+            </FormLabel>
+            <FormControl id="date-of-birth">
+                <FormLabel>Date Of Birth (YYYY-MM-DD) </FormLabel>
+                <Input placeholder="Date Of Birth" />
+            </FormControl>
+            <FormControl id="street-address-1">
+                <FormLabel>Street Address 1</FormLabel>
+                <Input placeholder="249 Phillip Street" />
+            </FormControl>
+            <FormControl id="street-address-2">
+                <FormLabel>Street Address 2</FormLabel>
+                <Input placeholder="284 Phillip Street" />
```

Comment:
```suggestion
                <Input placeholder="APT 20" />
```

Usually the address line 2 is for suite/apartment number!